frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -601,13 +571,18 @@ const ManagePolicyPage = ({
     ? teamPoliciesError
     : globalPoliciesError;
 
-  const policyResults = isAnyTeamSelected ? !!teamPolicies : !!globalPolicies;
+  const policyResults = isAnyTeamSelected
```

Comment:
This name implies that this is will be a collection of said results of some sort - can we call it `hasPolicyResults` or similar?

Reply:
true true, merging and iterating so I don't have to switch gears rn to get this in.